.github/workflows/claude.yml

```
@@ -1,5 +1,80 @@
 name: Claude Code
 
+# Define reusable allowed tools list under a legal node (e.g. env)
+env:
+  ALLOWED_TOOLS: &allowed-tools |
+    # Git operations
+    …
+  Bash(git add *)
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 8: could not find expected ':'

(syntax-check)